package.json

```diff
@@ -1,38 +1,44 @@
 {
-  "name": "eslint-plugin-netguru-ember",
-  "version": "1.6.5",
-  "description": "Eslint plugin for Netguru Ember Style Guide",
+  "name": "eslint-plugin-ember",
+  "version": "1.0.0",
+  "description": "Eslint plugin for Ember.js apps",
   "main": "index.js",
   "directories": {
     "rules": "rules",
     "test": "test"
   },
   "scripts": {
-    "test": "gulp test"
+    "test": "gulp test",
+    "start": "gulp watch"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/netguru/eslint-netguru-ember.git"
+    "url": "git+https://github.com/netguru/eslint-plugin-ember.git"
   },
   "keywords": [
     "eslint",
     "ember",
-    "netguru",
+    "ember.js",
     "plugin",
-    "styleguide"
+    "styleguide",
+    "rules"
   ],
   "author": "Michał Sajnóg (michal.sajnog@netguru.co)",
   "contributors": [
-    "Rafał Leszczyński (raf.leszcz@gmail.com)"
+    "Rafał Leszczyński (raf.leszcz@gmail.com)",
```

Comment:
This format seems invalid - see https://docs.npmjs.com/files/package.json#people-fields-author-contributors for example:
```
"Barney Rubble <b@rubble.com> (http://barnyrubble.tumblr.com/)"
```

BTW, seems like cool idea to add GitHub account links for all contributors here, just like in README 👍 .